src/components/box/box.spec.tsx

```diff
@@ -119,4 +120,17 @@ describe("Box", () => {
       wrapper
     );
   });
+
+  it.each<BoxProps["position"]>(["fixed", "sticky"])(
+    "sets zIndex to 1 when position is %s",
```

Comment:
Is it worth doing a test that covers the undefined path of this too? Might be overkill but I figured I'd ask the question anyway.

Reply:
I feel it's somewhat redundant to assert the zIndex is undefined when the position is undefined if that's what you mean?

Reply:
Not quite. I was referring to if the position css value is `absolute` for example then asserting that the zIndex is `undefined`. Just something that explores the `false` path of the ternary statement.

Reply:
Those paths are implicitly covered in other test cases but I think there's some value in having an explicit assertion so will add